src/ipfs/tests.rs

```diff
@@ -0,0 +1,122 @@
+use std::thread;
+use std::time::Duration;
+
+use colored::Colorize;
+use futures::executor::block_on;
+use serial_test::serial;
+
+use self::daemon::IpfsDaemon;
+use super::*;
+use crate::ipfs::Ipfs;
+use crate::utils::file_management;
+
+
+fn run_ipfs_test<T>(test: T) -> ()
+    where T: FnOnce(&IpfsDaemon) -> bool
+{
+    let ipfs = IpfsDaemon::default();
+    // for peer in PEER_ADDRS {
+    //     block_on(ipfs.connect_to(peer)).unwrap();
+    //     println!("Connected to peer! {}", peer);
+    // }
```

Comment:
Can this be removed? In general, we want to avoid leaving bits of commented out code in the codebase. 

This piece of code does look like it would be nice to have in verbose mode. If we do want to keep it around, perhaps we could uncomment it for now, and make a note to put it behind verbose mode when we have a way to do that.

Reply:
Oh actually, this is in a test. Verbose mode doesn't matter here. 😅 

I'd say lets keep the code or remove it depending on the feedback we want when running the test.